ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts

```diff
@@ -1713,6 +1713,15 @@ const triggerSet: TriggerSet<Data> = {
         'Usurper of Frost': 'Shiva-Mitron',
         'Oracle\'s Reflection': 'Spiegelbild des Orakels',
         'Ice Veil': 'Immerfrost-Kristall',
+        'Frozen Mirror': 'Eisspiegel',
+        'Holy Light': 'heilig\\[a\\] Licht',
+        'Crystal of Darkness': '[^:]+', // FIXME
```

Comment:
Should this be `'Crystal of Darkness': '[^\|]+'`? (and same for the other FIXMEs)